### PR TITLE
Install Tailwind CSS. Create files, and paste in the templates from the docs.

### PR DESCRIPTION
**Pull request by RossaMania**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -5,11 +5,14 @@
...
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
+    <%= stylesheet_link_tag "tailwind", "inter-font", "data-turbo-track": "reload" %>
 
     <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
   </head>
 
   <body>
-    <%= yield %>
+    <main class="container mx-auto mt-28 px-5 flex">
```
Comment (clettenberg) [inline app/views/layouts/application.html.erb:14]: Did you check to see if the styles applied to the `/trips` page? Can you post a screenshot

File: Procfile.dev [hunk 2]
```diff
@@ -0,0 +1,2 @@
+web: bin/rails server
```
Comment (clettenberg) [inline Procfile.dev:1]: Are you running a different command now to run both of these in your terminal?

File: bin/dev [hunk 3]
```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env sh
```
Comment (clettenberg) [inline bin/dev:1]: ah, is this the command you're running now to run the server? 

I think it's a good idea! Just checking.

**Review by clettenberg — CHANGES_REQUESTED** [review]
If you can add a screenshot of it working i'll approve!

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (RossaMania) [reply app/views/layouts/application.html.erb:14]: Happy to help! I have not applied any styles to the /trips page. I am actually working on that right now.

I am happy to post a screenshot once done!

Comment (clettenberg) [reply app/views/layouts/application.html.erb:14]: Since you added the tags to application.html.web, I think the trips page will get styling automatically

Comment (clettenberg) [reply app/views/layouts/application.html.erb:14]: So you should be able to fire up the app on this branch and visit /trips and see newer styles

Comment (clettenberg) [reply app/views/layouts/application.html.erb:14]: I could be wrong though lol

File: bin/dev (hunk 3 above)
Comment (RossaMania) [reply bin/dev:1]: image

Yes! The Procfile.dev and bin/dev files are installed automatically when Tailwind CSS is installed. This bin/dev file is similar to the "npm run dev" command in a React project. This should definitely come in handy when deploying the project to render.com. 

Once the server is running we should be able to start using Tailwind CSS in the application. I plan on making a homepage, and styling up the home page and trip page with Tailwind CSS. I figure I would keep it simple and start with the test boilerplate from the Tailwind docs.

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (RossaMania) [reply app/views/layouts/application.html.erb:14]: image

Happy to help! Here is a screenshot of the Tailwind CSS test code in my VS Code, and the "Hello World!" statement at http://localhost:3000/trips for your viewing pleasure.

**Review by clettenberg — APPROVED** [review]

**Pull request merged**